src/app/users/brokering/list-user-brokers.component.html

```diff
@@ -0,0 +1,64 @@
+<page-layout [ready]="data$ | async">
+  <ng-container *ngIf="data">
+    <page-content [mode]="'empty'" [heading]="i18n.brokers.title.list"
+      [mobileHeading]="i18n.brokers.mobileTitle.list"
+      [headingActions]="headingActions$ | async">
+    </page-content>
+    <results-layout [results]="data.brokers" [toLink]="toLink">
+      <mobile-result *mobileResult="let row">
```

Comment:
The `mobile-result`'s avatar and icon properties should be set, to show the broker image, or icon if no image.